modules/control-utility/src/main/java/org/apache/ignite/internal/commandline/cache/IdleVerify.java

```diff
@@ -341,6 +343,15 @@ private void cacheIdleVerifyV2(
         if (F.isEmpty(res.exceptions()))
             return;
 
+        Optional<Exception> inactiveE = res.exceptions().values()
+            .stream()
+            .filter(e -> X.hasCause(e, VerifyBackupPartitionsTaskV2.IDLE_VERIFY_ON_INACTIVE_CLUSTER_ERROR_MESSAGE,
+                IgniteException.class))
+            .findFirst();
+
+        if (inactiveE.isPresent())
+            throw new IllegalStateException(inactiveE.get().getMessage());
```

Comment:
Why only this particular exception treated as "illegal state", but other exceptions printed to "idle-verify...txt" file?